flask-redis-centos7-atomicapp/artifacts/kubernetes/00-service-redis.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: redis
+  labels:
+    name: redis
+spec:
+  ports:
+  - port: 6379
+    targetPort: 6379
```

Comment:
`targetPort` does not necessarily be a number, it could be a string as well, which identifies the port. Infact, it should be a word defining the port, like `redis-port`. The `targetPort` defined in redis pod manifest and `targetPort` in redis service manifest should match.

Check https://github.com/projectatomic/nulecule-library/blob/master/gitlab-centos7-atomicapp/artifacts/kubernetes/redis-service.json#L10 for reference.